src/InputWithLabel.js

```diff
@@ -0,0 +1,26 @@
+import React from'react';
+
+const InputWithLabel=({todoTitle, handleTitleChange, children}) => {
+    const inputRef = React.useRef(null);
+    React.useEffect (() =>{
+            inputRef.current.focus();
+        } 
+    );
+    
+    return (
+    <>
```

Comment:
empty tags should be indented one space beyond return since it is inside the return statement. label and input should be on matching indentations one space beyond the tags since they are both inside.